clients/drcachesim/tests/allasm-aarch64-prefetch-basic-counts.templatex

```diff
@@ -8,6 +8,8 @@ Total counts:
           81 total prefetches
      .* total data loads
      .* total data stores
+           0 total icache flushes
```

Comment:
All these 0's may lead to flaky tests if some ld.so or crt startup code on some system has a flush instruction: it may be better to put .* like the other entries (maybe even .* for both lines and not just the numbers, to also cover future basic_counts additions for rare things none of these tests will have so we don't have to go update all these files again).

Reply:
I put .* to match all numbers. 

Position of new entries may be different; adding .* here preemptively might not cover that.